tests/ses.test.js

```diff
@@ -6,6 +6,9 @@ const DB = require("../db/DB");
 const getSha1 = require("../sha1-utils");
 const ses = require("../controllers/ses");
 
+require("./resetDB");
```

Comment:
What happens if I add an `afterAll()` in this file now? Would it conflict with the one injected via resetDB.js?

Reply:
Hmm ... yes it would. I'll have to think about that.

Reply:
We could make ./resetDB.js super boring, and just have it export the `beforeAll()`, `beforeEach()`, and `afterAll()` functions.

Then in each test file, we do something similar to `beforeAll(resetDB.beforeAll)`.
If you want a custom `beforeEach()`, you just call the `resetDB.beforeEach();` and then call your custom code:

```js
beforeEach(async () => {
  await resetDB.beforeEach();
  customStuffIsFun();
});
```

A bit more verbose and boilerplate, but possibly more footgun protection and less side-effect-y.

Reply:
I'd like to think about the boilerplate-y code vs. foot-gun-y/side-effect-y code more before I change this. It may be like the `knexConfig` - the more we see it used/un-used the clearer it will be which approach is best.